feature_engine/transformation/boxcox.py

```diff
@@ -14,46 +14,6 @@ class BoxCoxTransformer(BaseNumericalTransformer):
     """
     The BoxCoxTransformer() applies the BoxCox transformation to numerical
     variables.
```

Comment:
@solegalli I deleted this because even when warnings are disabled there is still an error in this file and also the yeojohnson.py

I'm sure it's something obvious you can tweak now - I'm just not that familiar with sphinx